katakana-terminator.user.js

```diff
@@ -141,6 +149,20 @@ if (typeof GM_xmlhttpRequest === 'undefined' &&
     typeof GM === 'object' && typeof GM.xmlHttpRequest === 'function') {
     GM_xmlhttpRequest = GM.xmlHttpRequest;
 }
+if (typeof GM_addStyle === 'undefined') {
+    GM_addStyle = (aCss) => {
```

Comment:
May you please use the old school ES5 `function (aCss)` and `var` instead? So that we won't get angry emails from users that're still on old browsers (if there were any, I dunno).

Reply:
Actually those browsers won't even need this... But syntax error ¯\_(ツ)_/¯